control/control.openSUSE.xml

```diff
@@ -829,6 +827,11 @@ TODO: prepare disk here when it does not break space calculation (and remove bel
             <mode>update</mode>
             <stage>initial</stage>
             <modules config:type="list">
+                <!-- As soon as possible -->
```

Comment:
Hm, in SLES/SLED we configure network in manual upgrade, see https://github.com/yast/skelcd-control-SLES/blob/master/control/control.SLES.xml#L951, that is completely missing here...

I think should be added. Opinons? Or ideas what it could break?